src/main/java/org/zalando/nakadi/service/AuthorizationValidator.java

```diff
@@ -154,7 +154,8 @@ public void authorizeStreamRead(final EventType eventType) throws AccessDeniedEx
 
         final Resource resource = new EventTypeResource(eventType.getName(), eventType.getAuthorization());
         try {
-            if (!authorizationService.isAuthorized(AuthorizationService.Operation.READ, resource)) {
+            if (!authorizationService.isAuthorized(AuthorizationService.Operation.READ, resource)
```

Comment:
@lmontrieux But you have implemented it only for Low Level API and not for Subscription API. I thought you wanted to do vise versa.

Reply:
Ah, ok. You implemented it for both.

Reply:
I realised that implementing it here means that it will work for all endpoints that are protected with read permissions (for example, stats). It's already a hack solution and I'd rather not make it worse, I think it is fine to have it for LoLA as well, even though it is deprecated.

Reply:
Yes definitely. I think it doesn't make sense to spend additional effort to disable that for LoLA.